src/System.IO.FileSystem.Watcher.Polling/System/IO/PathToFileStateHashtable.cs

```diff
@@ -138,7 +132,7 @@ private int ComputeBucket(ReadOnlySpan<char> file)
         private void Resize()
         {
             // this is because sometimes we just need to garbade collect instead of increase size
-            var newSize = Math.Max(_count * 2, 4);
+            var newSize = Math.Max(Count * 2, 4);
```

Comment:
Also try to remove var when you're making changes. Var is ok for tests, but generally not for code.